rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -205,6 +318,98 @@ static void LIBUSB_CALL log_cb(libusb_context* /*ctx*/, enum libusb_log_level le
 }
 #endif
 
+void usb_handler_thread::perform_scan(bool force_scan)
+{
+	// look if any device which we could be interested in is actually connected
+	libusb_device** list = nullptr;
+	const ssize_t ndev   = libusb_get_device_list(ctx, &list);
+	std::set<uint64_t> seen_usb_devices;
+
+	if (ndev < 0)
+	{
+		sys_usbd.error("Failed to get device list: %s", libusb_error_name(static_cast<s32>(ndev)));
+		return;
+	}
+
+	// Skip device scan if we have the same amount of devices as last time
+	if (ndev != static_cast<ssize_t>(seen_usb_devices.size()) || force_scan)
```

Comment:
This seems to be wrong.
seen_usb_devices is always empty at this point.
And this will also never enter if ndev is 0.

Reply:
Ah yep, will fix that